lib/assets.js

```diff
@@ -43,7 +43,7 @@ TradeOfferManager.prototype._mapItemsToDescriptions = function(appid, contextid,
 			return new EconItem(item);
 		}
 
-		for (let i in cache[key]) {
+		for (const i in cache[key]) {
```

Comment:
In case there's still uncertainty about this, see [the definition of `ForDeclaration` in the spec](https://www.ecma-international.org/ecma-262/6.0/index.html#sec-iteration-statements).